src/compass/s1_geocode_metadata.py

```diff
@@ -140,6 +139,10 @@ def geocode_calibration_luts(geo_burst_h5, burst, cfg,
         Sentinel-1 burst SLC
     cfg: GeoRunConfig
         GeoRunConfig object with user runconfig options
+    dst_group_path:
+        xyz
+    item_dict:
+        abc
```

Comment:
```suggestion
    dst_group_path: str
        Path in HDF5 where geocode rasters will be placed
    item_dict: dict
        Dict containing item names and values to be geocoded
```
Sorry - forgot to correctly populate this in the PR